js/meetup/data.js

```diff
@@ -33,7 +72,7 @@ const meetups = [
         title: 'Comments in Rails Migrations and Databases',
         description: null,
         speaker: 'Suman Awal',
-        twitter_username: 'awalsuman43',
```

Comment:
I removed this as there was some issue with the twitter profle pic.

Reply:
looks like this problem exists with some other twitter profiles as well. 
Let's reverse this and tackle it in separate PR.

Reply:
Fixed it in https://github.com/RubyNepal/rubynepal.github.io/pull/424